### PR TITLE
Updated HICSS style

### PR DESCRIPTION
**Pull request by GeorgLink**
The style matches the template better:
http://hicss.hawaii.edu/wp-content/uploads/2016/09/hicss-formatspec-initialmanuscript.doc

Known issue:
- The comma after the title should be outside the quotation marks.

### CODE REVIEW

File: hawaii-international-conference-on-system-sciences-proceedings.csl [hunk 1]
```diff
@@ -36,10 +44,10 @@
   <macro name="title">
     <choose>
       <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
-        <text variable="title" font-style="italic" quotes="false"/>
+        <text variable="title" quotes="false" font-style="italic"/>
```
Comment (adam3smith) [inline hawaii-international-conference-on-system-sciences-proceedings.csl]: why the italics? Not seeing them in the styleguide.

File: hawaii-international-conference-on-system-sciences-proceedings.csl [hunk 2]
```diff
@@ -11,15 +12,22 @@
...
   </info>
+  <locale>
+    <style-options punctuation-in-quote="false"/>
+  </locale>
   <macro name="author">
-    <group suffix=".">
+    <group suffix=", ">
       <names variable="author">
-        <name initialize-with="." delimiter=", " and="text" name-as-sort-order="all" sort-separator=", "/>
+        <name and="text" delimiter-precedes-et-al="always" delimiter-precedes-last="always" initialize-with="." name-as-sort-order="first"/>
```
Comment (adam3smith) [inline hawaii-international-conference-on-system-sciences-proceedings.csl]: Do we have anything more than the templace? One of the examples corresponds to `name-as-sort-order="first"`, the other `="never"`

File: hawaii-international-conference-on-system-sciences-proceedings.csl [hunk 3]
```diff
@@ -49,13 +57,13 @@
     </date>
   </macro>
   <macro name="journal">
-    <group suffix=", " delimiter=" " font-style="italic">
-      <text variable="container-title"/>
-      <text variable="volume"/>
+    <group font-style="normal" suffix=", ">
+      <text variable="container-title" font-style="italic"/>
```
Comment (adam3smith) [inline hawaii-international-conference-on-system-sciences-proceedings.csl]: Same here -- any indication of italics somewhere other than the styleguide?

File: hawaii-international-conference-on-system-sciences-proceedings.csl (hunk 2 above)
Comment (GeorgLink) [reply hawaii-international-conference-on-system-sciences-proceedings.csl]: Unfortunately, HICSS does not have a good style guide and when I pull examples from last year the authors follow different conventions and get published. The option reflects what I believe to look the best, based on previous years, while still adhering to the limited style guide.

File: hawaii-international-conference-on-system-sciences-proceedings.csl (hunk 1 above)
Comment (GeorgLink) [reply hawaii-international-conference-on-system-sciences-proceedings.csl]: same as above

File: hawaii-international-conference-on-system-sciences-proceedings.csl (hunk 3 above)
Comment (GeorgLink) [reply hawaii-international-conference-on-system-sciences-proceedings.csl]: same as above

File: hawaii-international-conference-on-system-sciences-proceedings.csl (hunk 2 above)
Comment (GeorgLink) [reply hawaii-international-conference-on-system-sciences-proceedings.csl]: PS: I have a conditional accept with these settings and no comment was made about changing the style.

**Comment by adam3smith** [comment]
OK, let's go with this then. Thanks!

**Comment by GeorgLink** [comment]
Thank you!

**Comment by GeorgLink** [comment]
Update: The HICSS paper was accepted with this style. 👍

**Pull request merged**